specification/changeanalysis/resource-manager/Microsoft.ChangeAnalysis/preview/2020-04-01-preview/changeanalysis.json

```diff
@@ -41,7 +41,7 @@
           "default": {
             "description": "Error response describing why the operation failed",
             "schema": {
-              "$ref": "#/definitions/Error"
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
```

Comment:
The schema in the common type does not match old one. Which one is correct one the api returns?

Reply:
The error type service return is updated to support v2 - this was part of other private API work we did, which required for our service to support all the latest common types.